src/main/welcomeview/welcomeview.ts

```diff
@@ -709,22 +711,26 @@ export class WelcomeView {
   }
 
   private async _onEnvironmentListUpdated() {
+    const installMessage = `
+      <div class="warning-message">
+        Before you start, we need to set up your workspace. <a class="install-python-button" href="javascript:void(0);" onclick="sendMessageToMain('${EventTypeMain.InstallBundledPythonEnv}')">Get Started</a>
+      </div>
+    `;
     this._registry
       .getDefaultEnvironment()
       .then(() => {
-        this.enableLocalServerActions(true);
-        this.showNotification('', false);
+        // Check if the default environment is the bundled one
+        if (!this._registry.isDefaultEnvironmentBundled()) {
+          this.enableLocalServerActions(false);
+          this.showNotification(installMessage, false);
+        } else {
+          this.enableLocalServerActions(true);
+          this.showNotification('', false);
```

Comment:
Should we delete this line if it's an empty Notification? ... it looks like it's in the original code?